demisto_sdk/commands/validate/tests/BC_validators_test.py

```diff
@@ -727,3 +732,48 @@ def test_IsBreakingContextOutputBackwardsValidator_is_valid(
             for result, expected_msg in zip(results, expected_msgs)
         ]
     )
+
+
+def create_dummy_integration_with_context_path(
+    command_name: str, context_path: str
+) -> Integration:
+
+    dummy_integration = create_integration_object()
+    command = Command(name=command_name)
+    output = Output()
+    output.contextPath = context_path
+    command.outputs = [output]
+    dummy_integration.commands = [command]
+
+    return dummy_integration
+
+
+@pytest.mark.parametrize(
+    "old_context_path, new_context_path, len_expected_result",
+    [
+        pytest.param(
+            "test.test",
+            "test.test1",
+            1,
+            id="context path has been changed",
+        ),
+        pytest.param(
+            "test.test",
+            "test.test",
+            0,
+            id="valid integration",
+        ),
+    ],
+)
+def test_IsContextPathChangedValidator(
+    old_context_path, new_context_path, len_expected_result
+):
+    new_integration = create_dummy_integration_with_context_path(
+        command_name="command", context_path=new_context_path
+    )
+    old_integration = create_dummy_integration_with_context_path(
+        command_name="command", context_path=old_context_path
+    )
+    new_integration.old_base_content_object = old_integration
+    result = IsContextPathChangedValidator().is_valid(content_items=[new_integration])
+    assert len(result) == len_expected_result
```

Comment:
Let's discuss it offline

Reply:
was changed to one test which includes the two cases